WcaOnRails/app/models/result.rb

```diff
@@ -64,11 +63,11 @@ def country_iso2
   end
 
   def regional_single_record
-    regionalSingleRecord || ""
+    regionalSingleRecord
   end
 
   def regional_average_record
-    regionalAverageRecord || ""
+    regionalAverageRecord
   end
```

Comment:
Could this be implemented by virtue of Rails `alias_attribute`?

Reply:
> At the risk of overlooking something completely obvious here: Isn't the update query you suggested extremely dangerous? It overrides the columns if either one of the regional record markers is blank, which would in theory override a perfectly valid single `WR` just if the average WR regional record marker is set to an empty string...

Let's say it was a test to make sure you were paying attention and that I did not actually suggest to run this :sweat_smile: 

I've updated the original post with two separate queries which should be safe.